src/components/templates/Auditorium/components/SectionPreviews/SectionPreviews.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from "react";
+import { useHistory } from "react-router";
+import classNames from "classnames";
+
+import { AuditoriumSizes } from "types/auditorium";
+import { AuditoriumVenue } from "types/venues";
+
+import { useAuditoriumSections } from "hooks/auditoriumSections";
+import { useVenueId } from "hooks/useVenueId";
+
+import { SectionPreview } from "../SectionPreview";
+import { Video } from "../Video";
+
+import "./SectionPreviews.scss";
+
+export interface SectionPreviewsProps {
+  venue: AuditoriumVenue;
+}
+
+export const SectionPreviews: React.FC<SectionPreviewsProps> = ({ venue }) => {
+  const { iframeUrl } = venue;
+
+  const venueId = useVenueId();
+  const history = useHistory();
+  const { auditoriumSections } = useAuditoriumSections(venueId);
+
+  const sectionsAmount = auditoriumSections.length;
+
+  const auditoriumSize: AuditoriumSizes = useMemo(() => {
+    if (sectionsAmount <= 4) return AuditoriumSizes.SMALL;
+
+    if (sectionsAmount > 4 && sectionsAmount <= 10) {
+      return AuditoriumSizes.MEDIUM;
+    }
+
+    return AuditoriumSizes.LARGE;
+  }, [sectionsAmount]);
```

Comment:
This doesn't need a `useMemo` as it's basic logic and will run super quick anyway. You could extract this as a helper function if you want to 'clean it up' in a similar structure though. Something like:

```typescript
const chooseAuditoriumSize = (sectionCount: number): AuditoriumSize = { /** ..snip.. **/ }
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1251/commits/c7e60d97eac7ff85f121f5b5d8a480f8f16c3d70